pytest.importorskip("dgl") errors a lot

This keeps happening for some reason:

```
 ________ ERROR collecting openff/nagl/tests/molecule/dgl/test_batch.py _________
openff/nagl/tests/molecule/dgl/test_batch.py:7: in <module>
    dgl = pytest.importorskip("dgl")
../../../micromamba/envs/openff-nagl-test/lib/python3.10/site-packages/dgl/__init__.py:14: in <module>
    from .backend import backend_name, load_backend  # usort: skip
../../../micromamba/envs/openff-nagl-test/lib/python3.10/site-packages/dgl/backend/__init__.py:122: in <module>
    load_backend(get_preferred_backend())
../../../micromamba/envs/openff-nagl-test/lib/python3.10/site-packages/dgl/backend/__init__.py:107: in get_preferred_backend
    config_dict = json.load(config_file)
../../../micromamba/envs/openff-nagl-test/lib/python3.10/json/__init__.py:293: in load
    return loads(fp.read(),
../../../micromamba/envs/openff-nagl-test/lib/python3.10/json/__init__.py:346: in loads
    return _default_decoder.decode(s)
../../../micromamba/envs/openff-nagl-test/lib/python3.10/json/decoder.py:337: in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
../../../micromamba/envs/openff-nagl-test/lib/python3.10/json/decoder.py:355: in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
E   json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

--- a/.github/workflows/gh-ci.yaml
+++ b/.github/workflows/gh-ci.yaml
@@ -87,6 +87,7 @@ jobs:
     - name: Rewrite DGL config
       if: matrix.include-dgl == true
       run: |
+        mkdir -p ~/.dgl
         echo '{"backend": "pytorch"}' > ~/.dgl/config.json
 
     - name: Python information

--- a/.github/workflows/gh-ci.yaml
+++ b/.github/workflows/gh-ci.yaml
@@ -84,9 +84,10 @@ jobs:
       run: conda remove --force rdkit --yes || echo "rdkit not installed"
 
     # See https://github.com/openforcefield/openff-nagl/issues/103
-    - name: Remove DGL config
+    - name: Rewrite DGL config
       if: matrix.include-dgl == true
-      run: cat ~/.dgl/config.json || echo "no dgl config"
+      run: |
+        echo '{"backend": "pytorch"}' > ~/.dgl/config.json
 
     - name: Python information
       run: |

--- a/.github/workflows/gh-ci.yaml
+++ b/.github/workflows/gh-ci.yaml
@@ -83,6 +83,11 @@ jobs:
       if: matrix.include-rdkit == false
       run: conda remove --force rdkit --yes || echo "rdkit not installed"
 
+    # See https://github.com/openforcefield/openff-nagl/issues/103
+    - name: Remove DGL config
+      if: matrix.include-dgl == true
+      run: rm -rf ~/.dgl
+
     - name: Python information
       run: |
         which python

--- a/.github/workflows/gh-ci.yaml
+++ b/.github/workflows/gh-ci.yaml
@@ -86,7 +86,7 @@ jobs:
     # See https://github.com/openforcefield/openff-nagl/issues/103
     - name: Remove DGL config
       if: matrix.include-dgl == true
-      run: rm -rf ~/.dgl
+      run: cat ~/.dgl/config.json || echo "no dgl config"
 
     - name: Python information
       run: |

